examples/example.md

```diff
@@ -1,136 +1,160 @@
-# Crossplane Provider IONOS Cloud - Example (PoC)
+# Crossplane Provider IONOS Cloud POC
```

Comment:
I know that it was already there, but the `POC` naming is not suitable for this. I'd suggest the following: 
```suggestion
# Crossplane Provider IONOS Cloud Usage Example
```